api_compatibility_policy.md

```diff
@@ -90,7 +90,22 @@ For more information on support windows, see the [deprecations table](./docs/dep
 
 ## Feature Gates
 
-Stability levels of feature gates are independent from CRD apiVersions. Features enabled by API fields at different levels of stability can be enabled using the flag `enable-api-fields`:
+Stability levels of feature gates are independent from CRD apiVersions. 
+
+[TEP0138](https://github.com/tektoncd/community/blob/main/teps/0138-decouple-api-and-feature-versioning.md) has introduced per-feature flags for new API-driven features and the migration plan for `enable-api-fields`. Please refer to the table below for the API-driven features validation changes:
+
+| Releases               | Features                      | API-driven Features Validation     |
+| ---------------------- | ----------------------------- | ---------------------------------- |
+| Prior to v0.53.0       | All features                  | `enable-api-fields` **only** |
+| After v0.53.0        | [Existing features](#snapshot-of-existent-beta-and-alpha-api-driven-features-as-of-v0530httpsgithubcomtektoncdpipelinereleasestagv0530) till v0.53.0 | `enable-api-fields`            |
+| After v0.53.0        | New features after v0.53.0      |  Per-feature flags                 |
+| ...                    |                               |                                    |
+| Future release       | All [existing features](#snapshot-of-existent-beta-and-alpha-api-driven-features-as-of-v0530httpsgithubcomtektoncdpipelinereleasestagv0530) stabilized or deprecated | Sunset ~~`enable-api-fields`~~ |  
```

Comment:
"All existing features" will sound confusing in future, despite the link. 
I find the table a bit difficult to parse, what about this?

| Releases               | Global flag `enable-api-fields`                       | Per-feature flags     |
| ---------------------- | ----------------------------- | ---------------------------------- |
| Prior to v0.53.0       | All API features                  |  Alpha/beta  non-API features only |
| After v0.53.0        | [Alpha/beta features in v0.53.0](#snapshot-of-existent-beta-and-alpha-api-driven-features-as-of-v0530httpsgithubcomtektoncdpipelinereleasestagv0530) | Alpha/beta  non-API features and API features introduced after v0.53.0  |
| All [alpha/beta features in v0.53.0](#snapshot-of-existent-beta-and-alpha-api-driven-features-as-of-v0530httpsgithubcomtektoncdpipelinereleasestagv0530) become stable or are removed |    Sunset ~~`enable-api-fields`~~   |  All alpha/beta API and NON-API features |

Reply:
That makes sense! I made some minor changes to it because behavioural flags will retain their behaviour. (I also added a note below)

Reply:
Also updated the links at https://github.com/tektoncd/community/pull/1091.